src/Hl7.Fhir.Support.Poco/Rest/Http/HttpRequester.cs

```diff
@@ -0,0 +1,114 @@
+﻿/* 
+ * Copyright (c) 2014, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://raw.githubusercontent.com/FirelyTeam/fhir-net-api/master/LICENSE
+ */
+
+using Hl7.Fhir.Utility;
+using System;
+using System.IO.Compression;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Hl7.Fhir.Rest
+{
+    public class HttpRequester : IRequester, IDisposable
```

Comment:
I think this can be `internal`

Reply:
Is the `IRequester` interface used somewhere?

Reply:
Yes, it's used in the` BaseFhirClient`, and `HttpRequester` is used in `FhirHttpClient`. So we can't make it internal.